java/src/jmri/jmrit/display/panelEditor/configurexml/PanelEditorXml.java

```diff
@@ -177,28 +173,13 @@ public boolean load(Element shared, Element perNode) {
          value = true;
          panel.setShowCoordinates(value);
          */
-        value = true;
-        if ((a = shared.getAttribute("showtooltips")) != null && a.getValue().equals("no")) {
-            value = false;
-        }
+        value = (a = shared.getAttribute("showtooltips")) == null || !a.getValue().equals("no");
         panel.setAllShowToolTip(value);
-
-        value = true;
-        if ((a = shared.getAttribute("controlling")) != null && a.getValue().equals("no")) {
-            value = false;
-        }
+        value = (a = shared.getAttribute("controlling")) == null || !a.getValue().equals("no");
         panel.setAllControlling(value);
```

Comment:
JDOM2 has some nice methods to simplify this

http://www.jdom.org/docs/apidocs/org/jdom2/Element.html#getAttributeValue(java.lang.String,%20java.lang.String)

Reply:
thanks.  I hadn't actually looked at what those lines were doing, only the simplification of the logic statements.